src/pages/login/Login.jsx

```diff
@@ -23,10 +23,15 @@ export const Login = () => {
 
   const logInHandler = async (e) => {
     try {
-      e.preventDefault();
-      await dispatch(logIn(logInData)).unwrap();
-      navigate("/home");
-      toast("suceessfully login", { icon: "✔" });
+      if (logInData.email && logInData.password === "") {
```

Comment:
In Login Page, Redirecting to homepage even if the email and password are empty.